### PR TITLE
Adding a section about default queue permissions

### PR DESCRIPTION
**Pull request by hmemcpy**
@Particular/nservicebus @SimonCropp please review

### CODE REVIEW

**Comment by SzymonPobiega** [comment]
Looks good to me

File: nservicebus/msmq/transportconfig.md [hunk 1]
```diff
@@ -25,6 +25,11 @@ The main loops starts by subscribing to `PeekCompleted` event and calling the `B
 Because of historic reasons, the configuration for MSMQ transport has been coupled to general bus configuration in the previous versions of NServiceBus.
 
 
+### Queue permissions
+
+By default, queues are created with with the `Everyone` and `Anonymous Logon` permissions, in order to be able to send and receive messages without any additional configuration. This, however, means that anyone could send a message to a queue, leaving a potential security concern. If your organization requires it, set the appropriate permissions on a queue after its creation.
```
Comment (adamralph) [inline nservicebus/msmq/transportconfig.md]: Suggestion for alternate wording:

> By default, queues are created with `Everyone` and `Anonymous Logon` permissions to allow messages to be sent and received without additional configuration. Whilst initially convenient, this unrestricted access may be a security concern. If your organization requires it, you should set appropriate permissions on a queue after its creation.

Comment (hmemcpy) [inline nservicebus/msmq/transportconfig.md]: :+1:

File: nservicebus/msmq/transportconfig.md [hunk 2]
```diff
@@ -61,7 +66,7 @@ From V4 onwards the configuration for this mechanism is implemented in the `Tran
  * `MaxRetries` (default: `5`) defines how many times a message is tried to be processed before is is moved to the *error queue* or passed to the [Second-Level Retries, SLR](/nservicebus/errors/automatic-retries.md) mechanism.
  * `ErrorQueue` (default: `error`) sets the name of the queue where poison messages are sent to (including messages that failed *MaxRerties* number of times with SLR disabled and messages which cannot be processed at all, e.g. having unparsable or missing headers)
 
-In V3 some of these setting were available via `MsqmTransportConfig` section with following 
+In V3 some of these setting were available via `MsqmTransportConfig` section with following
```
Comment (danielmarbach) [inline nservicebus/msmq/transportconfig.md]: Are these changes necessary?

File: nservicebus/msmq/transportconfig.md [hunk 3]
```diff
@@ -61,7 +66,7 @@ From V4 onwards the configuration for this mechanism is implemented in the `Tran
...
 
  * In V3 the `ErrorQueue` (the queue where messages that fail a configured number of times) settings can be set both via the new `MessageForwardingInCaseOfFaultConfig ` section and the old `MsmqTransportConfig` section.
- * In V3 the `MaxRetries` as well as the throttling  (`NumberOfWorkerThreads`) settings can be set only via `MsmqTransportConfig` section.
\ No newline at end of file
+ * In V3 the `MaxRetries` as well as the throttling  (`NumberOfWorkerThreads`) settings can be set only via `MsmqTransportConfig` section.
```
Comment (danielmarbach) [inline nservicebus/msmq/transportconfig.md]: Are these changes necessary?

Comment (hmemcpy) [inline nservicebus/msmq/transportconfig.md]: @danielmarbach my editor (Atom) must have trimmed the trailing whitespace.
I don't see the harm in keeping them, but I'm not sure html generation will agree. I'll try reverting in the next commit.

**Comment by adamralph** [comment]
LGTM

**Comment by hmemcpy** [comment]
Can I push the button?

**Comment by adamralph** [comment]
Go for it :smile:

**Comment by hmemcpy** [comment]
Woohoo!
